src/http3server.cc

```diff
@@ -398,4 +402,17 @@ namespace quic
     }
   }
 
+  Napi::Value Http3ServerJS::port(const Napi::CallbackInfo &info)
```

Comment:
Please move the functionality to the server status update. Calls from js into this object are rather slow.